File: 0009-palindrome-number/0009-palindrome-number.java
class Solution 
{
    public boolean isPalindrome(int x) 
    {
        int rev,temp,rem;
        temp=x;
        rev=0;
        if(x<0)
        {
            return false;
        }
        while(x!=0)
        {
            rem=x%10;
            rev=rev*10+rem;
            x=x/10;
        }
        if(temp==rev)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

File: 0009-palindrome-number/0009-palindrome-number.c
bool isPalindrome(int x) 
{
    int rem;
    long long reverse=0;
    int temp=x;
    if(x>=0)
    {
        do
        {
            rem=x%10;
            reverse=reverse*10+rem;
            x=x/10;
        }while(x!=0);
        if(temp==reverse)
            return true;
        else
            return false;
    }
    else
        return false;
}